.circleci/config.yml

```diff
@@ -13,8 +13,28 @@ commands:
       - checkout
       - run: pip install --user tox
       - run: tox -e "<<parameters.version>>-sphinx{<<parameters.sphinx-version>>}"
+  run-build:
+    description: "Ensure built assets are up to date"
+    steps:
+      - checkout
+      - run: npm ci
+      - run: npm run build
+      - run:
+          name: Ensure built assets are up to date
+          command: |
+            if [[ `git status sphinx_rtd_theme/ --porcelain` ]]
+            then
+              echo "ERROR: assets are out of date. Make sure to run 'npm run build' on your branch."
+              git status --porcelain
```

Comment:
should we add the same path here?

Reply:
Makes sense! I forked this from an existing CircleCI orb, so didn't pay too much attention here.